lib/gis/file_name.c

```diff
@@ -44,9 +65,30 @@ char *G_file_name(char *path,
 /*!
   \brief Builds full path names to GIS misc data files
 
-  \param[out] path buffer to hold resultant full path to file
-  \param dir misc directory
-  \param element database element (eg, "cell", "cellhd", "vector", etc)
+  Paths to misc files are in form:
+  /home/user/GISDBASE/LOCATION/mapset/dir/name/element
+
+  path input buffer memory must be preallocated by caller.
+
+  C:
+  @code
+  char path[GPATH_MAX];
+  @endcode
```

Comment:
Unfinished example? Maybe some spaces needed too, but the Python example might be enough.